README.md

```diff
@@ -2,6 +2,93 @@
 
 Build images from the commandline in a convenient way.
 
+## Installation
+
+This project is under development right now and needs to be run via:
+```
+$ go run github.com/osbuild/image-builder-cli/cmd/image-builder@main
+```
+we plan to provide rpm packages as well.
+
+
+## Prerequisites
+
+Make sure to have the required `osbuild` rpms installed:
+```
+$ dnf install osbuild osbuild-depsolve-dnf
+```
+for the repositories you need `osbuild-composer` as well (this will
```

Comment:
I suggest adding `osbuild-composer` to the list of installed packages, and making this paragraph a note about why image-builder needs osbuild-composer and that this dep will go away in the future. Basically make it dead simple by default.

Reply:
Thanks, that makes sense, I added it now.